packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -668,29 +696,27 @@ export default class Meeting extends StatelessSparkPlugin {
   }
 
   /**
-   *
    * get local media streams based on options passed
-   * @exports
-   * @param {object} mediaDirection A configurable options object for joining a meeting
-   * @returns {Promise}
+   * @param {Object} mediaDirection A configurable options object for joining a meeting
+   * @returns {Promise} see #Media.getUserMedia
    * @memberof Meeting
    */
   getMediaStreams = (mediaDirection) => Media.getUserMedia(mediaDirection);
 
   /**
-   *
    * Specify joining via audio (option: pstn), video, screenshare
-   * @exports
-   * @param {object} options A configurable options object for joining a meeting
-   * @param {Object } options.resourceId pass the deviceId
+   * @param {Object} options A configurable options object for joining a meeting
+   * @param {Object} options.resourceId pass the deviceId
+   * @param {MediaStream} options.localStream
+   * @param {MediaStream} options.localShare
    * @returns {Promise}
    * @memberof Meeting
    */
   addMedia(options = {}) {
     const {localStream, localShare} = options;
     // eslint-disable-next-line no-warning-comments
     // TODO wire into default config. There's currently an issue with the stateless plugin or how we register
-    this.mediaProperties.setMediaDirection(Object.assign(this.config.mediaSettings, options.mediaSettings));
+    this.mediaProperties.setMediaDirection(process.env.NODE_ENV === 'test' ? {} : Object.assign(this.config.mediaSettings, options.mediaSettings));
```

Comment:
let's edit this with mockSpark instance so that meeting has a spark.meetings.config.mediaSettings stored due to StatelessSparkPlugin